executor/aggfuncs/window_func_test.go

```diff
@@ -111,6 +143,16 @@ func buildWindowTester(funcName string, tp byte, constantArg uint64, orderByCols
 	return pt
 }
 
+func buildWindowMemTester(funcName string, tp byte, constantArg uint64, numRows int, orderByCols int, allocMemDelta int64, updateMemDeltaGens updateMemDeltaGens, results ...interface{}) windowMemTest {
```

Comment:
we can remove `results`, it is useless here

Reply:
I have removed it from the code.